src/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  margin: 0 auto;
+}
+
+.searchForm {
+  margin-top: 20px;
+  position: relative;
+  border: none;
+}
+
+.searchInput {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: inherit;
+  color: #3d4e61;
+  font-weight: 800;
+  font-size: 16px;
+}
+
+.searchInputSmall {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.searchInput::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.searchIcon {
+  display: block;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+  border: none;
+  background-image: url(./images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-color: transparent;
+}
+
+.searchIconSmall {
```

Comment:
You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST form (`class="form"`), where you apply common form styles

```
.form {
  margin: 20px 0;
  border: solid 1px #e1e7ed;
  border-radius: 4px;
  height: 70px;
  position: relative;
  margin-top: 20px;
}
```

And then you apply `modifier` to the SECOND form (`class="form form--small"`) that have ONLY differences with base class form
```
.form--small {
  height: 42px;
  border-radius: 4px;
}
```
So in this way you follow DRY (Don't repeat yourself) principle
Apply this principe all over the styles